dashboard/src/app/modules/campaign/components/campaign-simulation-pane/campaign-simulation-pane.component.html

```diff
@@ -11,5 +20,5 @@ <h5>Trajets incités</h5>
     <h5>Incitations distribués</h5>
     <p>{{ state.amount }} €/pts</p>
   </div>
-
+</div>
 </div>
```

Comment:
Il n'y a pas un soucis d'indentation ici ?
```suggestion
</div> 
 
```